doc/source/catalog_generation.rst

```diff
@@ -23,67 +31,153 @@ It should also be stressed here that the point and segment photometry source lis
+correct for local small-scale overestimates and/or underestimates.  Both the 27 and 3 pixel
+settings are configurable variables for the user. 
+
+Once a background and RMS image are determined using this final technique, a preliminary 
+background-subtracted image is computed so it can be evaluated for the percentage of negative
+values in the illuminated portion of the image. If the percentage of negative values exceeds a
+configurable and defined threshold (default value 15%), the computation of the background and RMS image 
+from this
+algorithm are discarded.  Instead the background and RMS images computed using **Background Algorithm 2**,
+with the associated updates, are ultimately chosen as the images to use.
+It cannot be emphasized enough that a well-determined background measurement, leading to a good
+threshold definition, is very crucial for proper and successful source identification.
+
+Through-out this section variables have been mentioned which can be configured by the user.  The
+values used for these variables for generating the default catalogs are deemed to be the best for 
+the general situation, but users can tune these values to optimize for their own data. To this end, 
+users can adjust parameter values
+in the <instrument>_<detector>_catalog_generation_all.json files in the following path:
 /drizzlepac/pars/hap_pars/default_parameters/<instrument>/<detector>/.
 
-1.3: Source Identification with DAOStarFinder
-^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+1.4: Image Kernel
+^^^^^^^^^^^^^^^^^
+In an attempt to optimize the source detection for the specific image being processed,
+the software attempts to derive a custom image kernel based upon the data.
+The multi-filter detection image is analyzed to find an isolated, non-saturated  
+point source away from the edge of the image to use as a template for a source detection kernel.  
+If no suitable source is found, the algorithm falls back to the use of a two-dimensional Gaussian
+kernel based upon the supplied FWHM and the 
+`astropy.convolution.Gaussian2DKernel <https://docs.astropy.org/en/stable/api/astropy.convolution.Gaussian2DKernel.html>`_ 
+Astropy tool.
+
+
+2: Point (Aperture) Photometric Catalog Generation
+--------------------------------------------------
+
+2.1: Source Identification with DAOStarFinder
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 We use the `photutils.detection.DAOStarFinder <https://photutils.readthedocs.io/en/stable/api/photutils.detection.DAOStarFinder.html>`_ Astropy tool to identify sources in the background-subtracted
-multi-filter detection image. Regions flagged in the previously created bad pixel mask are ignored by
-DAOStarFinder. This algorithm works by identifying local brightness maxima with roughly gaussian
+multi-filter detection image. 
+This algorithm works by identifying local brightness maxima with roughly gaussian
```

Comment:
This would be where the background computed using the algorithm in the previous section gets applied to the science data to initialize point-source detection processing.

Reply:
Done and fix will be grouped with remaining fixes/suggestions.